dockerfiles/bitbucket-server/Dockerfile

```diff
@@ -33,39 +33,17 @@ ENV BITBUCKET_PASSWORD <password>
 ENV BITBUCKET your.bitbucket.server.hostname
 
 # The Bitbucket server API URL
-ENV BITBUCKET_API $BITBUCKET/rest/api/1.0
+# ENV BITBUCKET_API $BITBUCKET/rest/api/1.0
```

Comment:
`BITBUCKET` and `BITBUCKET_API` should either both be defined here or commented out.